lib/imgix/client.rb

```diff
@@ -60,6 +52,27 @@ def prefix
 
     private
 
+    def create_request(endpoint, img_url)
+      req = Net::HTTP::Post.new(endpoint.path)
```

Comment:
I'm probably dumb but thought I'd check - why is the request here just using the `.path`? Does that just end up requesting to `/api/v1/...`, which will fail?

Reply:
It's not you, it's ruby haha. But yeah, here's the original:

https://github.com/imgix/imgix-rb/blob/cb48a169d3845c4cf9c597a7aa1b14ecf331fb52/lib/imgix/client.rb#L46

What `create_request` does reflects how `Post.new` works. `Post.new` sets up the request object. Then `HTTP.new` is instantiated using:

https://github.com/imgix/imgix-rb/blob/cb48a169d3845c4cf9c597a7aa1b14ecf331fb52/lib/imgix/client.rb#L50

This sets up the http handle which requests the `req` object here:

https://github.com/imgix/imgix-rb/blob/cb48a169d3845c4cf9c597a7aa1b14ecf331fb52/lib/imgix/client.rb#L52

I had your exact thoughts on this, but it turns out `Net::HTTP` is just a little quirky. I've tested this code by making live purger requests on my own sources; if it's not too much I'd appreciate it if you could test that this works for you as well. Only if you want to though, [here's some boilerplate and instructions](https://gist.github.com/ericdeansanchez/14ff5320fc8e26d68ae49910a22b3c0e) that should help out.